src/Client.js

```diff
@@ -641,6 +641,36 @@ class Client extends EventEmitter {
         return ContactFactory.create(this, contact);
     }
 
+    /**
+     * Returns an object with information about the message
+     * @param {string} messageId
+     * @returns {Promise<object>} message information
+     */
+    async getMessageInfo(messageId) {
+        return await this.pupPage.evaluate(async messageId => {
+            const msgExist = await window.Store.MsgInfo._find(messageId);
```

Comment:
you seem to be missing some stuff in Injected.js, or is this something that we already put to Store?

Reply:
Please explain the question.
MsgInfo is already exist

Reply:
is this method working?